appveyor.yml

```diff
@@ -73,28 +72,11 @@ artifacts:
   - path: '*.Rcheck\**\*.Rout'
     name: Logs
 
-  - path: "ospsuite_*_ubuntu18.tar.gz"
+  - path: "ospsuite_*ubuntu18.tar.gz"
     name: Ubuntu18
 
-  - path: "ospsuite_*_centOS7.tar.gz"
+  - path: "ospsuite_*centOS7.tar.gz"
```

Comment:
those are not created fot his package. Does not make sense I believe

Reply:
Should I also remove Ubuntu? Not really needed.

Reply:
yes. We just need the two version of the package. The one for linux and the one for windows (tar.gz vs zip)
see reporting engine to see how this is created

https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/develop/appveyor.yml